brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -279,6 +278,32 @@ private String getParentId(NodeTemplate nodeTemplate) {
         return rootSpec;
     }
 
+    private void manageBrooklynPoliy(Map<String, EntitySpec<?>> specs,
```

Comment:
This method would be better named something like `setLocationsOnSpecs`. In particular I'd avoid the word "manage" because it has different connotations in Brooklyn.